### DISCUSSION TITLE
Textadept 11.2 beta 3

### DISCUSSION THREAD

**Discussion by orbitalquark** [orbitalquark/textadept#108] [General]
Textadept 11.2 beta 3 is released from https://github.com/orbitalquark/textadept/releases/tag/textadept_11.2_beta_3

Bugfixes:

* Allow "Replace All" for empty regex matches like '^' and '$'.
* Fixed display of secondary selections on Linux.
* Fixed instances of incorrect caret/selection placement when typing after clearing buffer text.

Changes:

* Allow syntax highlighting to be preserved in selected text for default themes.
* `textadept.editing.filter_through()` respects multiple and rectangular selections.
* Support "Replace All" in multiple and rectangular selection.

**Comment by sigzero** [comment]
For the normal editing stuff I do, I am not having any issues with B3.

**Comment by orbitalquark** [comment]
Great, thanks for the feedback!
